presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigUtils.java

```diff
@@ -74,4 +74,123 @@ public static void deployConfigFiles(File configTar, Path configDir)
             throw new PrestoManagerException("Failed to deploy config files", untarResult);
         }
     }
+
+    public static void addConfigFiles(Path configDir)
+            throws PrestoManagerException
+    {
+        try {
+            deleteDirectoryContents(configDir);
+            if (containsRegularFiles(DEFAULT_CONFIG_DIR)) {
+                copyDir(DEFAULT_CONFIG_DIR, configDir);
+                updateProperty(configDir.resolve("node.properties"), "node.id", randomUUID().toString());
```

Comment:
It would be better if we could have this node ID be included in the install request, so all of the IDs are generated by the Controller.